L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapShowerProducer.cc

```diff
@@ -15,7 +15,7 @@ void L1TMuonEndCapShowerProducer::produce(edm::Event& iEvent, const edm::EventSe
   // Create pointers to output products
   auto out_showers = std::make_unique<l1t::RegionalMuonShowerBxCollection>();
   out_showers->clear();
-  out_showers->setBXRange(0, 0);
+  out_showers->setBXRange(-2, 2);
```

Comment:
An even better solution could be to make this range configurable

Reply:
Ah, I agree with that. I should have made it like that from the start. I'll backport this as is for now, but I'll move this to configurable soon